.mega-linter.yml

```diff
@@ -0,0 +1,26 @@
+# Configuration file for MegaLinter
+# See all available variables at https://megalinter.io/configuration/ and in linters documentation
+
+APPLY_FIXES: all # all, none, or list of linter keys
+ENABLE: # If you use ENABLE variable, all other languages/formats/tooling-formats will be disabled by default
+  - JSON
+  - MARKDOWN
+  - RST
+
+ENABLE_LINTERS: # If you use ENABLE_LINTERS variable, all other linters will be disabled by default
+  - PYTHON_BLACK
+  # - PYTHON_FLAKE8
+
+# DISABLE:
+#   - COPYPASTE # Uncomment to disable checks of excessive copy-pastes
+#   - SPELL # Uncomment to disable checks of spelling mistakes
+#   - REPOSITORY
+SHOW_ELAPSED_TIME: true
+FILEIO_REPORTER: false
+# DISABLE_ERRORS: true # Uncomment if you want MegaLinter to detect errors but not block CI to pass
+
+PRE_COMMANDS:
+  - command: pip --version && python -V
+  - command: pip install -r requirements.txt
+    cwd: "workspace"   # Will be run at the root of the workspace (usually your repository root)
+  - command: pip install urllib3
```

Comment:
still need the new line at of the file here

Reply:
Fixed